documentation/book/con-custom-resources-status.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// assembly-custom-resources.adoc
+
+[id='con-custom-resources-status-{context}']
+= {ProductName} custom resource status
+
+The `status` property of a {ProductName}-specific custom resource is configured to capture the _current_ state of the resource and make that information available.
+
+Status information is useful for tracking progress related to a resource achieving its _desired_ state, providing the time and reason the state of the resource changed and details of events causing delays.
```

Comment:
```suggestion
Status information is useful for tracking progress related to a resource achieving its _desired_ state as defined by the `spec` property. The `status` provides the time and reason the state of the resource changed and details of events preventing or delaying operator from realising the desired state.
```

Reply:
Slight tweak to tom's edit:

"preventing or delaying operator from realising the desired state" > "preventing or delaying the Operator from realizing the desired state."